platforms/nuttx/src/px4/common/usr_hrt.cpp

```diff
@@ -122,7 +120,7 @@ void
 hrt_init(void)
 {
 	px4_register_shutdown_hook(hrt_request_stop);
-	g_usr_hrt_task = px4_task_spawn_cmd("usr_hrt", SCHED_DEFAULT, SCHED_PRIORITY_MAX, 1000, event_thread, NULL);
+	g_usr_hrt_task = px4_task_spawn_cmd("usr_hrt", SCHED_DEFAULT, SCHED_PRIORITY_MAX, 2048, event_thread, NULL);
```

Comment:
Here we could use PX4_STACK_ADJUSTED(1024). This would result 1024 bytes on 32-bit targets and 2048 on 64-bit targets.

Reply:
Can do